authors/settings.py

```diff
@@ -162,3 +162,10 @@
     import django_heroku
     django_heroku.settings(locals())
 
+# Email settings
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_USE_TLS = True
```

Comment:
this and the variable on line 169 should also be in the environment variables. Kindly work on  this ASAP so I can have this merged.

Reply:
Thanks @RonKbS, I have worked on it